inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/repository/DataProxyConfigRepository.java

```diff
@@ -163,53 +175,108 @@ private Map<String, List<InLongIdObject>> reloadInlongId() {
             if (groupId == null) {
                 continue;
             }
-            // choose topic
-            String groupTopic = groupIdObj.getTopic();
-            String streamTopic = streamIdObj.getTopic();
-            String finalTopic = null;
-            if (StringUtils.isEmpty(groupTopic)) {
-                // both empty then ignore
-                if (StringUtils.isEmpty(streamTopic)) {
-                    continue;
-                } else {
-                    finalTopic = streamTopic;
-                }
+            Map<String, String> groupParams = this.getExtParams(groupIdObj.getExtParams());
+            Map<String, String> streamParams = this.getExtParams(streamIdObj.getExtParams());
+            this.parseFirstTopic(groupIdObj, streamIdObj, groupParams, streamParams, inlongIdMap);
+            this.parseSecondTopic(groupIdObj, streamIdObj, groupParams, streamParams, inlongIdMap);
+        }
+        return inlongIdMap;
+    }
+
+    /**
+     * getExtParams
+     */
+    @SuppressWarnings("unchecked")
+    private Map<String, String> getExtParams(String extParams) {
+        // parse extparams
+        if (!StringUtils.isEmpty(extParams)) {
+            try {
+                Map<String, String> groupParams = gson.fromJson(extParams, HashMap.class);
+                return groupParams;
+            } catch (Exception e) {
+                LOGGER.error(e.getMessage(), e);
```

Comment:
Suggest adding more info in the log, such as:
```java
LOGGER.error("parse ext params from entity error:", e);
```

Reply:
fix it